addons/udes_stock/views/stock_location_views.xml

```diff
@@ -51,6 +51,14 @@
                 <attribute name="domain">[('usage', '!=', 'internal')]</attribute>
             </xpath>
 
+            <!-- Add message log to show archiving events. -->
```

Comment:
Super super nit: Could we make this comment more generic? As it isn't just archiving it will show.

Reply:
Oops, copypasta :blush:. Comment updated.